src/components/menu/index.tsx

```diff
@@ -17,12 +17,18 @@ interface MenuItem {
 export interface MenuProps {
   title: string;
   items: MenuItem[];
+  fontWeightTitle?: 'bold' | 'regular';
```

Comment:
Would it make sense to type it the same as values in the theme?

Reply:
Yes, I changed it! Thank you